ambry-rest/src/main/java/com.github.ambry.rest/NettyMetrics.java

```diff
@@ -107,6 +111,11 @@
   // NettyServer
   public final Histogram nettyServerShutdownTimeInMs;
   public final Histogram nettyServerStartTimeInMs;
+  // ConnectionStatsHandler
+  Counter connectionsConnectedCount;
```

Comment:
`public final` (the `public` is not really required but keep it for consistency)